GamesCommand.cs

```diff
@@ -33,8 +33,9 @@ public async Task MessageReceivedAsync(SocketMessage message)
         {
             if (message.Content.StartsWith("!addgame "))
             {
-                var game = message.Content.Substring("!addgame ".Length).ToLower();
-                await AddGame(message, game);
+	            // For some reason the Discord library made the mention sanitizer internal :c but this nbsp is what they do
+	            var game = message.Content.Substring("!addgame ".Length).ToLower().Replace("@", "@\u200B");
+	            await AddGame(message, game);
```

Comment:
can you fix the mixing of tabs and spaces here

Reply:
Sorry about that, I tried to check before but my IDE must have done something weird.